tests/ui/box_collection.rs

```diff
@@ -18,7 +18,7 @@ fn test_macro() {
     boxit!(Vec::new(), Vec<u8>);
 }
 
-fn test(foo: Box<Vec<bool>>) {}
+fn test1(foo: Box<Vec<bool>>) {}
 
 fn test2(foo: Box<dyn Fn(Vec<u32>)>) {
     // pass if #31 is fixed
```

Comment:
`    // pass if #31 is fixed`

Not sure what this means here. Is it referring to issue #31?

Reply:
That looks like the reason.

Reply:
Maybe we can remove that comment within some future commit.